analyzer/src/main/kotlin/managers/Cargo.kt

```diff
@@ -236,11 +236,21 @@ private fun checksumKeyOf(metadata: JsonNode): String {
 
 private fun extractCargoId(node: JsonNode) = node["id"].textValueOrEmpty()
 
-private fun extractDeclaredLicenses(node: JsonNode): SortedSet<String> =
-    node["license"].textValueOrEmpty().split('/')
+private fun extractDeclaredLicenses(node: JsonNode): SortedSet<String> {
+    val declaredLicenses = node["license"].textValueOrEmpty().split('/')
         .map { it.trim() }
         .filterTo(sortedSetOf()) { it.isNotEmpty() }
 
+    // Cargo allows to declare non-SPDX licenses only by referencing a license file. Avoid reporting an [OrtIssue] by
```

Comment:
Isn't the comment about avoiding an OrtIssue a bit misleading, as before this change no OrtIssue would have been created, or? It would only be created if whatever `node["license_file"]` was assigned as-is to `declaredLicenses`.

Reply:
Ah, I see the comment was taken from `Npm.kt`. However, there the case is slightly different, as "SEE LICENSE IN " might be part of the regular field for a declared license. So for Npm, the mapping is necessary to avoid an OrtIssue. But as Cargo has a dedicated field for that, that argumentation does not hold. I'd simply drop that sentence.

Reply:
I wanted to use the same comment as in `Npm` because it's the same concept.

Reply:
As I tried to explain, the "Avoid reporting an OrtIssue" part is *not* the same concept here. You're talking about suppressing an issue that would not have been there without your change.

Reply:
I only saw your first comment when I added my response. I have replaced the sentence with a better explanation.